packages/angular_devkit/core/README.md

```diff
@@ -56,4 +56,79 @@ export class CoreSchemaRegistry implements SchemaRegistry {
 
 # Utils
 
-# Virtual FS
\ No newline at end of file
+# Virtual FS
+
+# Workspaces
```

Comment:
Can you give a usage example? Adding a property to a build target, for instance.

Reply:
Done.